ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -19,6 +19,8 @@ let prototyped_of_field = function
       Some "22.27.0"
   | "host", "last_software_update" ->
       Some "22.20.0"
+  | "pool", "migration_compression" ->
+      Some "22.32.0"
```

Comment:
This should have `22.32.0-next` if the autogenerator works properly...

Reply:
It gave me 22.32.0-next the first time but after some changes only gave me the above